collect_stata/write_json.py

```diff
@@ -149,105 +166,107 @@ def stats_number(elem, file_csv):
     }
 
 
-def uni_statistics(elem, file_csv):
-    """Call function to generate statistics depending on the variable type
+def uni_statistics(elem, data):
+    """Call function to generate statistics depending on the variable type.
 
-    Input:
-    elem: dict
-    file_csv: pandas DataFrame
+    Args:
+        elem (dict): Contains information of one variable.
+        data (pandas.DataFrame): Datatable of imported data.
 
-    Output:
-    statistics: OrderedDict
+    Returns:
```

Comment:
```
    Returns:
        Statistics for either categorical, nominal or numerical variables.
```
Additonally, you would not break the explaining text like this.
If it were an args it should look like this:

```
    Args:
        statistics: Statistics for either categorical,
                    nominal or numerical variables.
```